android/src/com/mapswithme/maps/editor/EditorHostFragment.java

```diff
@@ -112,7 +110,7 @@ public void onSaveInstanceState(Bundle outState)
   protected void editMapObject()
   {
     mMode = Mode.MAP_OBJECT;
-    UiUtils.hide(mSearch);
+    ((SearchToolbarController) mToolbarController).showControls(false);
```

Comment:
У патента он такой, можно хелпер сделать который кастить будет, но имхо не нужно.
> On Mar 23, 2016, at 5:18 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/EditorHostFragment.java <https://github.com/mapsme/omim/pull/2421#discussion_r57165880>:
> 
> > @@ -112,7 +110,7 @@ public void onSaveInstanceState(Bundle outState)
> >    protected void editMapObject()
> >    {
> >      mMode = Mode.MAP_OBJECT;
> > -    UiUtils.hide(mSearch);
> > +    ((SearchToolbarController) mToolbarController).showControls(false);
> mToolbarController нельзя сразу сделать SearchToolbarController?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2421/files/c0e4402b8e98b0b9f926527d3e5c98651aa7fc3f..68707ffcc41d4cf532acb22cf42b2f2177f63389#r57165880>